index.d.ts

```diff
@@ -63,6 +76,7 @@ export interface AxiosRequestConfig {
   adapter?: AxiosAdapter;
   auth?: AxiosBasicCredentials;
   responseType?: ResponseType;
+  responseEncoding?: ResponseEncoding;
```

Comment:
@renanwilliam  its tyoe should be String so it should be -

responseEncoding?: String;